stylesheets/_component.progress-lists.scss

```diff
@@ -230,3 +230,48 @@
         z-index: 1;
     }
 }
+
+.progress-steps {
```

Comment:
do these need a lexicon example - I'm unsure what they are for?

Reply:
removed this as it was reimplemented by Nikos as `.stepped-progressbar` (which is the style being used by CMS Galaxies Designer)

Reply:
👍